features/user_login.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 'User logging'
+
+    An app user should be allowed the login based only on their roles and permissions
+
+    Scenario: A supervisor cannot login to stock app
+    Given I am a supervisor logging into "stock" app
+    And I enter the mobile number
+    Then I should not be allowed to login
+
+    Scenario: A reviewer cannot login to stock app
+    Given I am a reviewer logging into "stock" app
+    And I enter the mobile number
+    Then I should not be allowed to login
+
+    Scenario: A supervisor can login to admin app
+    Given I am a supervisor logging into "admin" app
+    And I enter the mobile number
+    And I enter the pin
+    Then I should be allowed to login
+
+    Scenario: A reviewer can login to admin app
+    Given I am a reviewer logging into "admin" app
+    And I enter the mobile number
+    And I enter the pin
+    Then I should be allowed to login
+
+    Scenario: A order_administrator can login to stock app
+    Given I am a order_administrator logging into "stock" app
+    And I enter the mobile number
+    And I enter the pin
+    Then I should be allowed to login
+
+    Scenario: A order_administrator cannot login to admin app
+    Given I am a order_administrator logging into "admin" app
+    And I enter the mobile number
+    Then I should not be allowed to login
```

Comment:
good to add cases for

- Stock fulfillment can login to Stock app
- Stock fulfillment cannot login to Admin app

etc etc

Reply:
Very happy to see cucumber tests :)

Although it does encourage the practice of having per-app code behaviours on the backend, which I think we should get rid of (cc @steveyken )

Reply:
It does, I'm not sure if there is a way to avoid that in this case. We want to prevent users from logging in to apps they don't have permission on. E.g. donor on Stock app. Can you see a better way?